### ISSUE TITLE
Draggable items on jacket

### ISSUE THREAD

**Issue by ofilippov** [din/simplify_jackets#6]
If I try to drag an item on my custom jacket (with mousedown and mousemove event listeners on this item) I will drag whole mini-player on screen instead.
We need a way to prevent player dragging somehow. Some mark on inside-draggable items for example.

**Comment by din** [comment]
You can implement your own dragging from scratch. You should use [Jacket options](https://github.com/mmth/simplify_jackets#jacketoptions) to completely disable dragging handled by Simplify and implement your own dragging routine from scratch. There are tons of examples of doing that, so, I am pretty sure, it's not a problem.

**Comment by ofilippov** [comment]
I see what you mean - will try to do it this way, thanks!

**Comment by din** [comment]
I'm not quite sure about making your jacket bounds equal screen bounds. In that case user wouldn't be able to drag&drop screen icons (jackets appear on top of desktop icons). But we can think about adding an option to stick jacket below every other element on screen, so you can maximise it on screen and implement your own drag&drop operations flawlessly.

**Comment by ofilippov** [comment]
Yeah, I removed that comment (about jacket with screen-size). It would be better to use small jacket with mouse listeners on it and call Simplify.setJacketFrame to change the position on screen. Looks like it should work.

**Comment by ofilippov** [comment]
Nope, doesn't work because of this: https://github.com/mmth/simplify_jackets/issues/8
